observability-kit-starter-hilla/pom.xml

```diff
@@ -9,7 +9,8 @@
         <version>2.1-SNAPSHOT</version>
     </parent>
 
-    <artifactId>observability-kit-starter-hilla</artifactId>
+    <groupId>dev.hilla</groupId>
```

Comment:
This brings a problem, in maven central we cannot deploy at once with different namespaces, it needs some work in the parent module for having profiles to include certain modules and also changes in CI for adding steps for first deploying to com.vaadin and then dev.hilla

Reply:
It's the same we have for SSO Kit: I added a profile to exclude the Hilla module build unless the `hilla` property is true.